Shared/sdk/CVector.h

```diff
@@ -26,11 +26,15 @@ class CVector
     static constexpr float FLOAT_EPSILON = 0.0001f;
 
 public:
-    float fX = 0.0f;
-    float fY = 0.0f;
-    float fZ = 0.0f;
+    float fX;
+    float fY;
+    float fZ;
 
-    constexpr CVector() = default;
+    struct NoInit{};
+
+    CVector(NoInit) {}
```

Comment:
Whats the point behind it?

Reply:
This is the call to a special constructor within vector's components initialization. For the sake of the performance.

Reply:
See for example https://github.com/rheit/zdbsp/blob/master/tarray.h or https://docs.unrealengine.com/4.27/en-US/API/Runtime/Core/UObject/FNameEntryId/